### PR TITLE
Replace practiSe by practiCe

### PR DESCRIPTION
**Pull request by fitn**
"practice" is the noun
"to practice" is the verb
"you're practising" is the only usage of "s" instead of "c"

### CODE REVIEW

**Comment by fitn** [comment]
It's for you @rybus ;)

File: cookbook/import_export/clean-csv-file-during-product-import.rst [hunk 1]
```diff
@@ -90,7 +90,7 @@ Then we declare this new array converter service in ``array_converters.yml``.
 
     You can notice here that we use the `Decorator Pattern`_ by injecting the default array converter in our own class.
 
-    The big advantage of this practise is to decouple your custom code from the PIM code, for instance, if in the future, an extra dependency is injected in the constructor of the default array converter, your code will not be impacted.
+    The big advantage of this practice is to decouple your custom code from the PIM code, for instance, if in the future, an extra dependency is injected in the constructor of the default array converter, your code will not be impacted.
```
Comment (rybus) [inline cookbook/import_export/clean-csv-file-during-product-import.rst]: Too long sentence, split it to facilitate reading.

Comment (fitn) [inline cookbook/import_export/clean-csv-file-during-product-import.rst]: Sorry but not in this PR. I would not rewrite the entire stuff just fix these typos.

File: contributing/best_practices.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Best Practises
+Best Practices
```
Comment (nidup) [inline contributing/best_practices.rst]: yes finaly!

**Comment by nidup** [comment]
GTM

**Pull request merged**